client/db/src/lib.rs

```diff
@@ -391,15 +391,23 @@ impl std::fmt::Display for DatabaseSource {
 	}
 }
 
-pub(crate) mod columns {
+/// Column constants.
+pub mod columns {
+	/// Meta
 	pub const META: u32 = crate::utils::COLUMN_META;
+	/// State
 	pub const STATE: u32 = 1;
+	/// State meta
 	pub const STATE_META: u32 = 2;
 	/// maps hashes to lookup keys and numbers to canon hashes.
 	pub const KEY_LOOKUP: u32 = 3;
+	/// Header
 	pub const HEADER: u32 = 4;
+	/// Body
 	pub const BODY: u32 = 5;
+	/// Justifications
```

Comment:
maybe we should be some actually useful doc comments?

Reply:
Yea I have no idea what to put here. Linter forced me to put some since I had to make it public.

Reply:
Talked with @cheme and he was more comfortable when I expose the column through `expose_db` as no not break the encapsulation.  
Done now